people-and-planet-ai/land-cover-classification/predict_batch.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This does model predictions for different locations at a range of years."""
+
+import csv
+from typing import Iterable, List, NamedTuple, Optional, Sequence, Tuple
```

Comment:
Seems like at the moment, `typing.NamedTuple` still has an advantage over `collections.namedtuple`

```suggestion
from collections.abc import Iterable, Sequence
from typing import List, NamedTuple, Optional
```